bool clear            #clear current list of segmented objects before adding new ones
bool useMapFrame      #if true, segment objects with respect to the map frame rather than the robot's base_footprint frame
bool segmentOnRobot   #if true, perform segmentation on the CARL's base plate, use plane detection otherwise
---
sensor_msgs/PointCloud2[] objects   #the segmented point clouds
